lib/cassia/requests/switch_autoselect.rb

```diff
@@ -0,0 +1,37 @@
+module Cassia
+  module Requests
+    class SwitchAutoSelect
```

Comment:
one minor thing, the class should be `SwitchAutoselect` since the file is names `switch_autoselect.rb`. If it was `switch_auto_select.rb` then the class would be `SwitchAutoSelect`. This is just ruby semantics